views/postgres/create.erb

```diff
@@ -174,6 +174,38 @@
                 </div>
               </div>
 
+              <div class="col-span-full">
+                <div class="space-y-2">
+                  <label for="size" class="text-sm font-medium leading-6 text-gray-900">Version</label>
+                  <fieldset class="radio-small-cards" id="size-radios">
+                    <legend class="sr-only">Version</legend>
+                    <div class="grid gap-3 grid-cols-1 md:grid-cols-2 xl:grid-cols-3">
+                      <% Option::PostgresVersionOptions.select { _1.family == @flavor }.map(&:version).each_with_index do |version, idx| %>
```

Comment:
What if we keep `Option::PostgresVersionOptions` as hash and select versions by O(1) with a key instead of select and map the list?